packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -330,8 +331,8 @@ class AndroidGradleBuilder implements AndroidBuilder {
       AndroidStudioJavaGradleConflictMigration(_logger,
           project: project.android,
           androidStudio: _androidStudio,
-          java: globals.java)
-      ,
+          java: globals.java),
+      MinSdkVersionMigration(project.android, _logger),
```

Comment:
Nit:weird spacing here

Reply:
Is this spacing weird? The migration is on the same line as the previous migration (`AndroidStudioJavaGradleConflictMigration`, above).

Reply:
you are correct I missed that this is a list